### PR TITLE
feat: make photo fields more prominent

### PR DESCRIPTION
**Pull request by PeterBaker0**
# feat: make photo fields more prominent

Moves the 'label' text from the button itself to a <\h3\> component. Moves the helper text into a \<p\> element which helps with size and spacing. The button is adorned with a camera icon, and uses a contained style which makes it more visible. 

@stevecassidy worth considering if this change will negatively impact any existing notebooks which rely on the button text being the label - I didn't really personally think this was very helpful as the button always does the same thing, in any notebook, it 'takes a photo'. 

image

## Checklist

- [x] I have confirmed all commits have been signed.
- [x] I have added JSDoc style comments to any new functions or classes.
- [x] Relevant documentation such as READMEs, guides, and class comments are updated.

### CODE REVIEW

File: app/src/gui/fields/TakePhoto.tsx [hunk 1]
```diff
@@ -287,49 +300,57 @@ export class TakePhoto extends React.Component<
...
+            <Button
+              variant="contained"
+              color="primary"
+              fullWidth={isMobile ? true : false}
+              onClick={this.takePhoto}
+            >
+              Take photo
+              <span style={{width: 10}} />
+              <CameraAltIcon />
+            </Button>
```
Comment (PeterBaker0) [inline app/src/gui/fields/TakePhoto.tsx:324]: This is the only part that actually changed - FYI

File: app/src/gui/fields/TakePhoto.tsx [hunk 2]
```diff
@@ -18,23 +18,36 @@
...
+ */
+function MediaQueryWrapper({
+  children,
+}: {
+  children: (isMobile: boolean) => React.ReactNode;
+}) {
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+  return children(isMobile);
+}
+
```
Comment (stevecassidy) [inline app/src/gui/fields/TakePhoto.tsx:50]: Is it worth defining this component here just for a single use, seems a lot of overhead for a boolean flag, line 47 is all you need.

File: app/src/gui/fields/TakePhoto.tsx (hunk 1 above)
Comment (stevecassidy) [reply app/src/gui/fields/TakePhoto.tsx:324]: Happy with the layout change.

**Review by stevecassidy — APPROVED** [review]
Ok but question the need for introducing a new component.

**Review by luke-mcfarlane-rocketlab — APPROVED** [review]
LGTM

File: app/src/gui/fields/TakePhoto.tsx (hunk 2 above)
Comment (PeterBaker0) [reply app/src/gui/fields/TakePhoto.tsx:50]: As discussed - cannot use custom hooks in class fields. Rewrite later.

**Pull request merged**